iree/turbine/kernel/wave/constraints.py

```diff
@@ -132,19 +134,112 @@ def mma_matrix_shapes(self, mma_type: Optional[MMAType]) -> tuple[int]:
                 return (32, 32, 8)
             case (
                 MMAType.F32_16x16x32_F8
+                | MMAType.F32_16x16x32_K8_F16
                 | MMAType.F32_16x16x32_K4_F8
                 | MMAType.I32_16x16x32_I8
             ):
                 return (16, 16, 32)
             case (
                 MMAType.F32_32x32x16_F8
+                | MMAType.F32_32x32x16_K8_F16
                 | MMAType.F32_32x32x16_K4_F8
                 | MMAType.I32_32x32x16_I8
             ):
                 return (32, 32, 16)
             case _:
                 return ()
 
+    def mma_index_offset(self, mma_type: Optional[MMAType]):
+        lane = self.linearized_thread_id % self.threads_per_wave
```

Comment:
minor: use lane as input argument instead of recomputing it? Also seems like the mma_type check is also being done outside this function so not needed inside?

Reply:
I am using it from within this because we are using this functionality from the decompose_vmma_ops pass as well to be able to search and replace the Index Expression of the virtual mma with the origina native one during decomposition

Reply:
makes sense, thanks